index.bs

```diff
@@ -1163,6 +1163,16 @@ Meeting Scheduling and Announcements</h4>
 	Shorter notice for a meeting is allowed
 	provided that there are no objections from group participants.
 
+<h4 id="meeting-minutes">
+Meeting Minutes</h4>
+
+	Groups are expected to take and retain <dfn>minutes</dfn> of their meetings,
+	and <em class="rfc2119">must</em> record in particular
+	any official [=group decisions=] made during the meeting discussions.
+	Precise details of the discussion leading to such decisions
```

Comment:
Remove "Precised details".

Reply:
Do you mean to suggest removing the word "precise" only? Removing both words would make the sentence ungrammatical.

Reply:
'records of the discussion…'